### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by tiagoGottardo**

### CODE REVIEW

File: simulation/simulation.py [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
     self.financing_value = financing_value
     self.installments_number = installments_number
     self.age = age
 
   def run(self):
       try:
-        bank = Banks[self.bank].value
-      except ValueError:
+        bank = Banks[self.bank.upper()].value
+      except KeyError:
```
Comment (samueldurantes) [inline simulation/simulation.py:13]: Why did you use `KeyError`?

File: Makefile [hunk 2]
```diff
@@ -2,3 +2,7 @@
 run:
 	@echo "Running the program..."
 	python -m flask --app api.main run
+test:
```
Comment (samueldurantes) [inline Makefile:5]: Add `.PHONY` in this target.

**Review by samueldurantes — APPROVED** [review]

File: simulation/simulation.py (hunk 1 above)
Comment (tiagoGottardo) [reply simulation/simulation.py:13]: if the key does not match any member name in enum, python throws KeyError.

File: Makefile (hunk 2 above)
Comment (tiagoGottardo) [reply Makefile:5]: ok

File: simulation/simulation.py (hunk 1 above)
Comment (samueldurantes) [reply simulation/simulation.py:13]: Fair enough.

**Review by samueldurantes — APPROVED** [review]

**Pull request merged**